eva-lib/src/main/java/uk/ac/ebi/eva/lib/entities/Taxonomy.java

```diff
@@ -46,13 +46,21 @@
 })
 @NamedNativeQueries({
         @NamedNativeQuery(
-                name = "Taxonomy.getSpecies",
+                name = "Taxonomy.getBrowsableSpecies",
                 query = "select distinct(assembly.*), taxonomy.* " +
                         "from assembly left join browsable_file bf on assembly.assembly_set_id=bf.assembly_set_id " +
+                        " and assembly.assembly_accession = bf.loaded_assembly " +
```

Comment:
This additional join condition is required because the same assembly-set-id can have different assembly accessions (usually different versions of the same assembly). Browsable status is per-assembly and not per-assembly-set.

Reply:
Bonus points, `L_crocea_1.0` and `SL2.40` are no longer listed twice.